example/lib/slides/footer_slide.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_deck/flutter_deck.dart';
+
+class FooterSlide extends FlutterDeckSlideWidget {
+  const FooterSlide()
+      : super(
+          configuration: const FlutterDeckSlideConfiguration(
+            footer: FlutterDeckFooterConfiguration(
+              showSlideNumbers: true,
+              widget: Row(
+                children: [
+                  Icon(Icons.home),
+                  SizedBox(width: 8),
+                  Text('This is a custom footer with icon and text'),
+                ],
+              ),
+            ),
+            route: '/footer-slide',
+            header: FlutterDeckHeaderConfiguration(
+              title: 'Custom footer slide',
+            ),
+          ),
+        );
+
+  @override
+  FlutterDeckSlide build(BuildContext context) {
+    return FlutterDeckSlide.blank(
+      builder: (context) => Center(
+        child: Text(
+          'Based on the configuration, this template renders a header and a '
```

Comment:
Maybe it's worth explaining different footer configuration options instead of providing the same text as in the blank slide example?